CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 0.19.3.1 (2015-12-03)
+- Add a flag `--no-systemd` to avoid using systemd with docker.
```

Comment:
Can you also have the issue # here? The release note you put here is pretty confusion.